python/gcp_client/CONTRIBUTING.md

```diff
@@ -0,0 +1,176 @@
+# Guidance on how to contribute
```

Comment:
We may want to try using `git`'s symlink tracking for these files. I can see down the road versions of this document changing. Just thinking for long term maintainability/laziness.

Reply:
Done. Seems to play fine with setuptools